win/PyTango_VS9/PyTango.vsprops

```diff
@@ -90,30 +90,30 @@
         Description="[PyTango]: Starting python $(WinSetupArgs)"
     />
 
-    <UserMacro Name="PyTangoBaseFile" Value="_PyTango" />
+    <UserMacro Name="PyTangoBaseFile" Value="_tango" />
     
-    <UserMacro Name="TangoMajorVer" Value="8" />
-    <UserMacro Name="TangoMinorVer" Value="1" />
+    <UserMacro Name="TangoMajorVer" Value="9" />
+    <UserMacro Name="TangoMinorVer" Value="2" />
     <UserMacro Name="TangoPatchVer" Value="2" />
     <UserMacro Name="TangoVer" Value="$(TangoMajorVer).$(TangoMinorVer).$(TangoPatchVer)" />

```

Comment:
the current Tango release for Windows is indeed 9.2.2, so maybe we have to update PyTango version to match it ?